modules/tensor_mechanics/doc/content/source/materials/lagrangian/ComputeSimoHughesJ2PlasticityStress.md

```diff
@@ -0,0 +1,87 @@
+# ComputeSimoHughesJ2PlasticityStress
+
+!syntax description /Materials/ComputeSimoHughesJ2PlasticityStress
```

Comment:
Can you add a reference to the paper (I assume there is a paper besides the book)

Reply:
Added